contracts/insured-vesting-v1/InsuredVestingV1.sol

```diff
@@ -8,9 +8,12 @@ import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
 contract InsuredVestingV1 is Ownable {
     using SafeERC20 for IERC20;
 
+    uint256 TOKEN_RATE_PRECISION = 1e20;
+
+    // TODO(Audit) comment - rename "PROJECT_TOKEN" (FUNDING_TOKEN for insured)
     IERC20 public immutable USDC;
     IERC20 public immutable XCTD;
-    uint256 public immutable USDC_TO_XCTD_RATE;
+    uint256 public immutable XCTD_TO_USDC_RATE; // TODO(audit) rename to PROJECT_TOKEN_TO_FUNDING_TOKEN_RATE
```

Comment:
intentionally not renaming yet to generic token names, will be modified in a subsequent PR